docs/includes/apiargs-MongoDBCollection-method-aggregate-option.yaml

```diff
@@ -2,72 +2,81 @@ arg_name: option
 name: allowDiskUse
 type: boolean
 description: |
-  Enables writing to temporary files. When set to true, aggregation
-  stages can write data to the ``_tmp`` sub-directory in the dbPath
-  directory. The default is ``false``.
+  Enables writing to temporary files. When set to ``true``, aggregation stages
+  can write data to the ``_tmp`` sub-directory in the ``dbPath`` directory. The
+  default is ``false``.
 interface: phpmethod
 operation: MongoDB\\Collection::aggregate
 optional: true
-position: 1
 ---
 arg_name: option
 name: batchSize
 type: integer
 description: |
-  The number of documents to return per batch
+  Specifies the initial batch size for the cursor. A batchSize of ``0`` means an
+  empty first batch and is useful for quickly returning a cursor or failure
+  message without doing significant server-side work.
+
+  .. note::
+
+     This is not supported for inline aggregation results (i.e. ``useCursor``
+     option is ``false`` or the server version is < 2.6).
 interface: phpmethod
 operation: MongoDB\\Collection::aggregate
 optional: true
-position: 2
 ---
 source:
+  file: apiargs-MongoDBCollection-common-option.yaml
   ref: bypassDocumentValidation
-  file: apiargs-common-option.yaml
-operation: MongoDB\\Collection::aggregate
-position: 5
+post: |
+  This only applies when using the :pipeline:`$out` stage.
```

Comment:
Is this just a byproduct of me rendering the PHPLIB docs separately from the main manual (on my local machine), or do you expect this would persist even in the final builds. I might as well just use ``$out`` if this will have no effect in the deployed version.